.github/workflows/phpstan.yml

```diff
@@ -19,6 +19,7 @@ jobs:
           - "8.0"
           - "8.1"
           - "8.2"
+          - "8.3"
```

Comment:
This is too daring 👴🏻 for viktor.

Reply:
I agree. I just want to see how the matrix explodes with this additional version, which we need in the near future. will revert afterwards

Reply:
```
tomasvotruba/unused-public 0.2.0.72 requires php ^7.2 || 8.0.*
 -> your php version (8.3.0-dev) does not satisfy that requirement.
```

Reply:
I guess this will fix it: https://github.com/staabm/phpstan-dba/pull/619